starforge/forge/wheels.py

```diff
@@ -214,10 +214,14 @@ def bdist_wheel(self, output=None, uid=-1, gid=-1):
 
         chdir(join(build, root))
 
-        if self.wheel_config.insert_setuptools or self.image.plat_specific or not check_setup():
+        insert_setuptools = self.wheel_config.insert_setuptools
+        if not insert_setuptools and insert_setuptools is not False:
```

Comment:
Is this equivalent to the simpler `if insert_setuptools is None:` ?

Reply:
Yeah, not sure what gymnastics I engaged in to get this.